xcode/Subconscious/Shared/Components/Common/ExpandAlignedLeadingViewModifier.swift

```diff
@@ -13,7 +13,7 @@ struct ExpandAlignLeadingViewModifier: ViewModifier {
     func body(content: Content) -> some View {
         HStack(spacing: 0) {
             content
-            Spacer()
+            Spacer(minLength: 0)
```

Comment:
It turns out that the spacing problem was not caused by the transclude failing to fill horizontal space. Rather, it was the spacer in this modifier taking up a minimum amount of space.

Setting the minimum to zero fixes the issue.